src/crypto.js

```diff
@@ -1,8 +1,15 @@
 import { ERRORS, checkObject, MasqError } from './errors'
 
-const genRandomBuffer = (len = 16) => {
+const _checkEncodingFormat = (format) => {
+  if (format !== 'hex' && format !== 'base64') throw new MasqError(ERRORS.INVALID_ENCODING_FORMAT)
+}
+
+const genRandomBuffer = (len = 16, encodingFormat) => {
```

Comment:
The return output is inconsistent: it either return a Buffer, or a string.

I think we should keep this function simple by just generating a random buffer. The user os this function will call `toString(encoding)` if we wants